ark/utils/segmentation_utils.py

```diff
@@ -33,6 +35,56 @@ def find_nuclear_mask_id(nuc_segmentation_mask, cell_coords):
     return nuclear_mask_id
 
 
+def split_large_nuclei(cell_segmentation_mask, nuc_segmentation_mask, cell_ids):
+    """Splits nuclei that are bigger than the corresponding cell into multiple pieces
+
+    Args:
+        cell_segmentation_mask (np.ndarray):
+            mask of cell segmentations
+        nuc_segmentation_mask (np.ndarray):
+            mask of nuclear segmentations
+        cell_ids (np.ndarray):
+            the unique cells in the segmentation mask
+
+    Returns:
+        np.ndarray:
+            modified nuclear segmentation mask
+    """
+
+    nuc_mask_modified = np.copy(nuc_segmentation_mask)
+    max_nuc_id = np.max(nuc_segmentation_mask)
+
+    cell_props = pd.DataFrame(regionprops_table(cell_segmentation_mask,
+                                                properties=['label', 'coords']))
+
+    for cell in cell_ids:
+        coords = cell_props.loc[cell_props['label'] == cell, 'coords'].values[0]
+
+        nuc_id = find_nuclear_mask_id(nuc_segmentation_mask=nuc_segmentation_mask,
+                                      cell_coords=coords)
+
+        if nuc_id is None:
+            # no nucleus found within this cell
+            pass
+        else:
+            # figure out if nuclear label is completely contained within cell label
+            cell_vals = nuc_segmentation_mask[tuple(coords.T)]
+            nuc_count = np.sum(cell_vals == nuc_id)
+
+            nuc_mask = nuc_segmentation_mask == nuc_id
+            if nuc_count == np.sum(nuc_mask):
+                # nucleus is contained within the cell
+                pass
+            else:
```

Comment:
you were probs doing this for better readability during dev, but now these
```
if condition:
    pass
else:
    # buncha code
```
can just be
```
if not condition:
    # buncha code
```